src/pages/Components/Notification/previews/notification-states.preview.scss

```diff
@@ -0,0 +1,20 @@
+@import 'settings-tools/_all-settings';
+@import 'components/_c.links';
+@import 'components/_c.notification';
+
+body {
+  padding-bottom: $mu600;
+}
+
+.example {
+  margin: $mu150;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
+
+.example__mc-notification {
+  margin-top: $mu150;
+  width: 38%;
```

Comment:
why this width ?

Reply:
Actually the purpose here is only display the notifications side by side. Regarding responsiveness, according to Gael, the Notification pattern will have no responsiveness info, and It will, by default, takes as much space as available by its parent, hence responsiveness will be handled by the user.